### DISCUSSION TITLE
Algae-based and Mycelium-based economy

### DISCUSSION THREAD

**Discussion by mokun** [mars-sim/mars-sim#443] [Ideas]
@bevans2000 @larsnaesbye @Urwumpe ,

I watched [this](https://www.youtube.com/watch?v=oy9NRjZQK10) with regard to the most recent effort in commercializing algae-based *footwear* products into everyday use. 

Previously, I also watched [this](https://youtu.be/cApVVuuqLFY?list=PLnTSM-ORSgi4dFnLD9622FK77atWtQVv7) and get fascinated in some of the research and development done on Mycelium Fungus to replace plastic in commercial applications.

I suppose it's safe to speculate on how the future Mars colony would not be repeating the same mistakes as we did on Earth and would readily adopt environmentally safe material in the products they make. 

So shall we start modeling how to grow algae and fungus ? 

What do you see as the first few manufacturing processes that make use of them ?

**Comment by mokun** [comment]
See projects, journals and articles such as 
1. [Microalgae as bioreactor for bioplastic production](https://www.researchgate.net/publication/51724760_Microalgae_as_bioreactor_for_bioplastic_production)
2. [AlgaePARC](https://www.wur.nl/en/Value-Creation-Cooperation/AlgaePARC.htm)
3. [Cyanobacteria for bioplastic production](https://www.researchgate.net/publication/321713391_Cyanobacteria_for_bioplastic_production)
4. [4th International Conference on Sustainable Bioplastics](https://www.omicsonline.org/conferences-list/methods-of-production-of-bioplastics)
5. [Bioplastic Made from Algae](https://bioplasticsnews.com/2014/02/24/bioplastic-made-from-algae/)

**Comment by larsnaesbye** [comment]
I have personally been interested in mycoleather for belts, wallets etc. but have no concrete experience with it.

**Reply by larsnaesbye** [reply]
E.g. https://boltthreads.com/

**Reply by mokun** [reply]
@bevans2000 @larsnaesbye 

I just watched [this](https://www.youtube.com/watch?v=jI2LC3WTryw) about the use of mushroom root, aka mycelium.

It's fascinating to know that mycelium can be used as excellent packaging material. It's an excellent textile material for clothings, pants, shoes and accessories.

It can be made artificial bacons with meat-like textures, more healthy than processed meat and a great culinary addition for Mars settlers.

It consumes minimal water and little energy input when growing mycelium, with seemingly waste products. 

It's more environmental than vinyl and polyurethane-based leather since it's biodegradable, replacing plastics and foams.

It can be used as insulating and prefab panels, wall claddings, and fire-retardant bricks.

We already had the idea of growing one species of mushroom in a mushroom containment box.

Now we just need to expand that to growing trays of mycelium in a storage building.

We could tweak the Farming class to GrowingFungi class to suit its need.

**Comment by Urwumpe** [comment]
I think the processes to produce methane from algae could also be interesting in this context.

**Comment by bevans2000** [comment]
To grow algae & fungi could we reuse the existing `Fishery` & `Farming` function in a `Building`?

**Reply by mokun** [reply]
Since one should be growing algae/fungi in a separate pond or bioreactor, should they be a separate building function ? Not in a greenhouse/fish farm ?

See this new [thread](https://github.com/mars-sim/mars-sim/issues/986).

**Comment by mokun** [comment]
- See snapshot below for the 7 major categories of plastic (recycling) in the U.S.. 
- In the U.S., only 5% of plastics were actually recycled. Most are mechanically processed, which involves physical processes such as grinding, washing, and regranulating to return used plastic to its solid form.
- It makes sense for future Mars colonies to utilize Mycelium instead of industrial plastic in order to avoid repeating the costly mistake of filling the ocean with microplastic.

![Screenshot 2025-06-30 at 9 49 41 AM](https://github.com/user-attachments/assets/4e2da3b3-c155-4b62-988a-18569db5d5be)

**Reply by bevans2000** [reply]
Ok, so how do we want to model this? A new Function with a lifecycle similar to Fishing?
Some Manufacturing Processes to create goods from the raw material.
Also a Salvage Process to turn the Goods back into raw material

**Comment by mokun** [comment]
> Ok, so how do we want to model this? A new Function with a lifecycle similar to Fishing?

We do have `AlgaeFarming` that extends `Function`. However, should growing fungi be the same as algae ?

Also, we haven't taken enough advantage of having algae at this point. 

So far, algae pond produces `Spirulina` as an edible amount resource and that's it.

Algae should have a lot of usage. It can even absorb harmful UV, etc.
